data/campaigns/Son_Of_The_Black_Eye/scenarios/04_The_Siege_of_Barag_Gor.cfg

```diff
@@ -495,7 +495,7 @@
 
         [message]
             speaker=Jetto
-            message= _ "The Barag Gór council sent me to assassinate the elvish leaders. Unfortunately, I was captured. But now I am free and the elves shall feel my wrath!"
+            message= _ "The Barag Gór Council sent me to assassinate the elvish leaders. Unfortunately, I was captured. But now I am free and the elves shall feel my wrath!"
```

Comment:
Same thing here.

Reply:
Precedent:
https://github.com/wesnoth/wesnoth/blob/6c78d376fe8f4d6b62d6392bef738e500854851d/data/campaigns/Son_Of_The_Black_Eye/scenarios/01_End_of_Peace.cfg#L246
and at least 15 other instances of 'Orcish Council' or 'Great Council' throughout the campaign.